newIDE/app/src/InstancesEditor/index.js

```diff
@@ -167,9 +167,12 @@ export default class InstancesEditor extends Component<Props> {
       if (this.keyboardShortcuts.shouldZoom()) {
         this.zoomOnCursorBy(-event.deltaY / 5000);
       } else if (this.keyboardShortcuts.shouldScrollHorizontally()) {
-        this.viewPosition.scrollBy(-event.deltaY / 10, 0);
+        this.viewPosition.scrollBy(-event.deltaY / 5, 0);
```

Comment:
Is it normal not to use the zoomFactor here?

Reply:
I had not tested this behavior as it was never triggered with my tests.
But now I realise this happens when the user press "alt" and then scroll vertically -> trigger a horizontal scroll.
(I didn't know this was a common scrolling feature 🤔 )
I guess it should apply the same logic as well!

Reply:
I guess same logic yeah! 
This was adapted from software doing the same, especially useful for traditional mouses/no trackpad which don't have a horizontal scroll.